src/components/Dropdown/Dropdown.spec.tsx

```diff
@@ -0,0 +1,124 @@
+    ],
+];
+
+const LARGE_ITEMS = [
+    [
+        {
+            id: "5",
+            title: "Large",
+            subtitle: "Subtitle",
+            size: Size.Large,
+        },
+        {
+            id: "6",
+            title: "Large warning",
+            subtitle: "Subtitle",
+            size: Size.Large,
+            warning: true,
+        },
+    ],
+    [
+        {
+            id: "7",
+            title: "Large disabled",
+            subtitle: "Subtitle",
+            size: Size.Large,
+            disabled: true,
+        },
+        {
+            id: "8",
+            title: "Large warning disabled",
+            subtitle: "Subtitle",
+            size: Size.Large,
+            warning: true,
+            disabled: true,
+        },
+    ],
+];
+
+type Props = {
+    items: MenuItem[][];
+    size: Size;
+    placeholder?: string;
+    initialActiveId?: string;
+};
+
+const Component: FC<Props> = ({ items, size, placeholder = "", initialActiveId = "" }) => {
+    const [activeItemId, setActiveItemId] = useState(initialActiveId);
+    return (
+        <Dropdown
+            activeItemId={activeItemId}
+            onChange={setActiveItemId}
+            menuItems={items}
+            placeholder={placeholder}
+            size={size}
+        />
+    );
+};
+
+describe("Dropdown Component", () => {
+    it("renders with placeholder", () => {
+        mount(<Component items={SMALL_ITEMS} size={Size.Small} placeholder="Select item" />);
+        cy.get(MENU_ITEM_TEXT_ID).first().contains("Select item");
+        cy.get(DROPDOWN_TRIGGER_ID).should("have.class", css.inactive);
+        cy.get(DROPDOWN_TRIGGER_ID).click();
+        cy.get(MENU_ITEM_TEXT_ID).should("have.length", 5);
+    });
+    it("renders with initial active item", () => {
+        mount(<Component items={SMALL_ITEMS} size={Size.Small} initialActiveId={SMALL_ITEMS[0][0].id} />);
+        cy.get(MENU_ITEM_TEXT_ID).first().contains("Small");
+        cy.get(DROPDOWN_TRIGGER_ID).should("not.have.class", css.inactive);
+    });
+    it("changes selection on click", () => {
+        mount(<Component items={LARGE_ITEMS} size={Size.Large} initialActiveId={LARGE_ITEMS[0][0].id} />);
+        cy.get(DROPDOWN_TRIGGER_ID).should("have.class", css.large);
+        cy.get(MENU_ITEM_TEXT_ID).first().contains("Large");
+        cy.get(DROPDOWN_TRIGGER_ID).click();
+        cy.get(MENU_ITEM_TEXT_ID).eq(2).click();
+        cy.get(MENU_ITEM_TEXT_ID).first().contains("Large warning");
+    });
+    it("renders divider", () => {
+        mount(<Component items={LARGE_ITEMS} size={Size.Large} initialActiveId={LARGE_ITEMS[0][0].id} />);
+        cy.get(DROPDOWN_TRIGGER_ID).click();
+        cy.get(DROPDOWN_MENU_ID).children().as("itemBlocks");
+        cy.get("@itemBlocks").should("have.length", 2);
+        cy.get("@itemBlocks").first().should("have.css", "border-bottom", BORDER_STYLE);
+        cy.get("@itemBlocks").last().should("not.have.css", "border-bottom", BORDER_STYLE);
```

Comment:
can't we just for the prop and ignore the style itself?

Reply:
I tried but somehow it only works when specifying a style 🤷🏽‍♀️